cluster-up-example

```diff
@@ -1,14 +1,10 @@
 [Nodes:children]
 master
-core
-mbaas
+macos
```

Comment:
yes, that's right. I finally understand what `ansible_connection=local` means. Apparently it should be only used for deploy to the local machine itself. For some reason I thought it means using the `local ssh`....